www/userguide/userguide.md

```diff
@@ -0,0 +1,126 @@
+automatically_. 
+
+\subsection run_diff Compare runs
+Calculates difference between two analyses of the project, showing which bugs
+have been fixed and which are newly introduced.
+
+\subsection run_delete Delete runs
+You can delete multiple runs by selecting them and clicking on the Delete
+button. It will remove the run and all related data from the database.
+
+\subsection run_order Reorder runs
+It is possible to change the order of the runs by clicking on a cell at header
+of the run list table. For example you can sort the run list by the number of
+bugs or the run name.
+
+\section report_overview Analysis results
+If you select a run at the [list of runs](#list_of_runs) view, you get to this
+page. This page lists the analysis result for the given run.
+ 
+![Reports](images/userguide/reports.png)
+
+\subsection severity Severity levels
+We are mapping checker names to different severity levels:
+Icon  | Level name | Meaning |Checkers
+:-------------: | ------------- | ------------- | -------------
+<span class="customIcon severity-unspecified"></span>  | Unspecified | **TODO** | **TODO** |
+<span class="customIcon severity-style"></span>  | Style | **TODO** | _modernize-raw-string-literal, modernize-use-auto, etc._ |
+<span class="customIcon severity-low"></span>  | Low | **TODO** | _deadcode.DeadStores, misc-unused-parameters, etc._ |
+<span class="customIcon severity-medium"></span>  | Medium | **TODO** | _unix.Malloc, core.uninitialized.Assign, etc._ |
+<span class="customIcon severity-high"></span>  | High | **TODO** | _core.DivideZero, core.NullDereference, cplusplus.NewDelete, etc._ |
+<span class="customIcon severity-critical"></span>  | Critical | **TODO** | **TODO** |
+
+\section bug_view Bug view
+At this page you can navigate between reports and check the errors what
+CodeChecker found.
+
+This page has four main parts:
+- [Report navigation tree](#report_navigation_tree)
+- [Button pane](#button_pane)
+- [Bug path view](#bug_path_view)
+- [Comments](#comment)
+
+![Report view](images/userguide/report_view.png)
+
+\subsection report_navigation_tree Report navigation tree
+Report Navigation Tree shows the found reports at the currently opened file. The
+reports are grouped by the severity level.
+You can navigate between them by clicking on a node in the tree.
+
+![Report navigation tree](images/userguide/report_navigation_tree.png)
+
+\subsection button_pane Button pane
+Button Pane contains several items which helps you to change or get some
+property of the currently opened report. 
+![Button pane](images/userguide/button_pane.png)
+
+\subsubsection checker_documentation Show documentation
+Show Documentation button shows the documentation of the actual checker which
+identified by the currently opened report.
+![Checker documentation](images/userguide/checker_documentation.png)
+
+\subsubsection review_status Review status
+Reports can be assigned a review status of Unreviewed, Confirmed bug, False
```

Comment:
Reports or bugs can be assigned a review status (detection status, comment, etc.)? If this information belongs to a bug then what is the connection between the bugs and reports? (unique mode)